File: android/src/com/kplusfarm/kotlinwar/android/AndroidLauncher.java
package com.kplusfarm.kotlinwar.android;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
import com.kplusfarm.kotlinwar.KotlinWar;

public class AndroidLauncher extends AndroidApplication {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
        initialize(new KotlinWar(), config);
    }
}

File: android/src/com/kplusfarm/kotlinwar/android/AndroidLauncher.kt
package com.kplusfarm.kotlinwar.android

import android.os.Bundle

import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
import com.kplusfarm.kotlinwar.KotlinWar

public class AndroidLauncher : AndroidApplication() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        val config = AndroidApplicationConfiguration()
        initialize(KotlinWar(), config)
    }
}
